packages/react-app-scaffolder/app/templates/graphql/src/core/private-route-wrapper.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react'
+import { useReapitConnect } from '@reapit/connect-session'
+import { Loader, Section, FlexContainerResponsive, AppNavContainer, FlexContainerBasic } from '@reapit/elements'
+import Menu from '@/components/ui/menu'
+import { reapitConnectBrowserSession } from '@/core/connect-session'
+import { useLocation, Redirect } from 'react-router'
+import { ApolloProvider } from '@apollo/react-hooks'
+import getClient from '@/graphql/client'
+
+const { Suspense } = React
+
+export type PrivateRouteWrapperProps = {}
+
+export const PrivateRouteWrapper: React.FunctionComponent<PrivateRouteWrapperProps> = ({ children }) => {
+  const { connectSession, connectInternalRedirect } = useReapitConnect(reapitConnectBrowserSession)
+  const location = useLocation()
+  const currentUri = `${location.pathname}${location.search}`
+  const accessToken = connectSession?.accessToken || ''
+
+  if (!connectSession) {
+    return null
+  }
+
```

Comment:
I think there is lack of isRoot case when redirect. Please refer below
```
 const isRoot = connectInternalRedirect === '/?' || connectInternalRedirect === '/' || window.location.pathname === '/'

  if (!connectSession) {
    return null
  }

  if (isRoot) {
    return <Redirect to={Routes.APPROVALS} />
  }

  if (connectInternalRedirect && currentUri !== connectInternalRedirect) {
    return <Redirect to={connectInternalRedirect} />
  }
```

Reply:
If I add the code like this
<img width="876" alt="Screen Shot 2020-09-03 at 18 16 58" src="https://user-images.githubusercontent.com/26746668/92108575-d5f9e400-ee11-11ea-8e0b-0681c138151a.png">
It loads a blank page when I start the app, do you have any idea ?